Formula/htmlq.rb

```diff
@@ -0,0 +1,18 @@
+class Htmlq < Formula
+  desc "Uses CSS selectors to extract bits content from HTML files"
+  homepage "https://github.com/mgdm/htmlq"
+  url "https://github.com/mgdm/htmlq/archive/0cb3b720451f85710b6c68894c39d521ca0d9b15.tar.gz"
+  sha256 "13bce9d1620bc8abd8fa5e408d76e67bc33b86d2640dc8753b6a8e119739b2aa"
+  version "0.1.1"
+  license "MIT"
+  head "https://github.com/mgdm/htmlq.git", branch: "master", revision: "0cb3b720451f85710b6c68894c39d521ca0d9b15"
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "#{bin}/htmlq", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Sorry I'm not a contributor of this project. 
I'll ask of developer.

Reply:
If you know how to use `htmlq`, that should be enough to write a test. Here's more from our documentation: https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula